### PR TITLE
include the whitelisted repositories on different organization than --only-org

### PR DESCRIPTION
**Pull request by droslean**
/cc @openshift/test-platform @stlaz 

Signed-off-by: Nikolaos Moraitis <nmoraiti@redhat.com>

### CODE REVIEW

File: cmd/ci-operator-config-mirror/main.go [hunk 1]
```diff
@@ -118,8 +117,11 @@ func main() {
 		}
 
 		if len(o.onlyOrg) > 0 && repoInfo.Org != o.onlyOrg {
-			logger.Warnf("Skipping... This repo doesn't belong in %s organization", o.onlyOrg)
-			return nil
+			if !o.WhitelistConfig.IsWhitelisted(repoInfo) {
+				logger.Warnf("Skipping... This repo doesn't belong in %s organization", o.onlyOrg)
```
Comment (jmguzik) [inline cmd/ci-operator-config-mirror/main.go:121]:
```suggestion
				logger.Warnf("Skipping... This repo doesn't belong to %s organization", o.onlyOrg)
```

Comment (droslean) [reply cmd/ci-operator-config-mirror/main.go]: addressed

**Pull request merged**